File: app/src/main/java/com/levibostian/androidblanky/db/models/OtherSampleModel.java
package com.levibostian.androidblanky.db.models;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;
import io.realm.annotations.Required;

public class OtherSampleModel extends RealmObject {

    @PrimaryKey public long id;
    @Required public String first_name;
    @Required public String last_name;
    @Required public String username;

    public OtherSampleModel() {
    }

    public String getFirstAndLastName() {
        return first_name.concat(" ").concat(last_name);
    }

}

File: app/src/main/java/com/levibostian/androidblanky/db/models/OtherSampleModel.kt
package com.levibostian.androidblanky.db.models

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import io.realm.annotations.Required

class OtherSampleModel : RealmObject() {

    @PrimaryKey var id: Long = 0
    @Required var first_name: String = ""
    @Required var last_name: String = ""
    @Required var username: String = ""

    fun firstLastName(): String = first_name + " " + last_name

}